clusters/build-clusters/02_cluster/installation_params.md

```diff
@@ -0,0 +1,52 @@
+# Build02: installation parameters
+
+## [Configuring a GCP project](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html)
+
+Create a new project on GCP.
```

Comment:
I will ask team-dpp for help on this, as we did this for `openshift-ci-infra` on aws for `build01`.

`openshift-ci-infra` on GCP is taken is for api.ci.
So does `openshift-build-farm` sound a good name?
(I did not put `build02` in the name because we could hold buildN in the project)

Reply:
Maybe "openshift-ci-build-farm"? Not a string opinion either way though.

Reply:
`openshift-ci-build-infra`?

Reply:
I will give our preference on this to DPP.
- openshift-ci-build-infra
- openshift-ci-build-farm

(@alvaroaleman 's candidate is in the 2nd because it is Not a str~i~ong opinion. ^_^)